spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -1046,11 +1078,31 @@ def _banner_default(self):
 
     def _kernel_restarted_message(self, died=True):
         msg = _("Kernel died, restarting") if died else _("Kernel restarting")
+
+        if died and self.kernel_manager is None:
+            # The kernel might never restart, show position of fault file
+            msg += (
+                "\nCrash file located at: "
+                + self.kernel_handler.fault_filename()
```

Comment:
```suggestion
            msg += (
                "\n" + _("Its crash file is located at:)" + " "
                + self.kernel_handler.fault_filename()
```

This makes the new message you added translatable.